tests/geometry/test_intermediate.py

```diff
@@ -0,0 +1,162 @@
+"""
+Unit Test for geometry.intermediate problems
+"""
+
+from unittest import TestCase
+
+from geometry.intermediate import Coordinate, LineSegment, do_line_segments_intersect, \
+    could_line_segment_contain_coordinate, calculate_point_triplet_orientation
+
+
+class TestLineSegmentsIntersection(TestCase):
+    """
+    Unit Test for identifying if two line segments intersect
+    """
+    def test_proper_line_segments_intersect(self):
+        """
+        Tests that two proper line segments which intersect, returns True
+        """
+        line_segment_1_start = Coordinate(-1, 0)
+        line_segment_1_end = Coordinate(1, 0)
+        line_segment_1 = LineSegment(line_segment_1_start, line_segment_1_end)
```

Comment:
I would just re-write all these to be:
```python
line_segment_1 = LineSegment(Coordinate(-1,0), Coordinate(1, 0))
```
My reasoning is that you'll have less code lines, and less code to read means less stuff to keep in your head.
This way it makes it more explicit. But it also makes assignments that aren't necessary.
It's all a matter of preference.